### PR TITLE
Add GPDB PITR tests to GPDB CI

### PR DESCRIPTION
**Pull request by roicos**
- This is a new container test which is separate from ICW
   because putting it in the ICW would cause unnecessary backups
   of the entire ICW workload.

### CODE REVIEW

File: concourse/pipelines/templates/gpdb-tpl.yml [hunk 1]
```diff
@@ -227,6 +227,7 @@ groups:
   - icw_planner_icproxy_centos7
   - icw_gporca_ictcp_centos7
   - unit_tests_gporca_centos7
+  - gpdb_pirt_centos7
```
Comment (jimmyyih) [inline concourse/pipelines/templates/gpdb-tpl.yml:230]: s/gpdb_pirt_centos7/gpdb_pitr_centos7/ for here and the rest of the file.

**Review by jimmyyih — COMMENTED** [review]
Only the template file was updated.  You need to generate the pipeline (e.g. `gen_pipeline.py --prod`) and add the updated generated pipeline to the patch.

File: concourse/scripts/gpdb_pitr.bash [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+						\$(cat "\${diff_file}")
+
+					FEOF
+			fi
+		    done
+		    exit 1
+		}
+		source /usr/local/greenplum-db-devel/greenplum_path.sh
+		cd "\${1}/gpdb_src"
+		make
```
Comment (jimmyyih) [inline concourse/scripts/gpdb_pitr.bash:33]: Is this `make` required?  Seems like it would unnecessarily compile the code.

Comment (roicos) [reply concourse/scripts/gpdb_pitr.bash]: We needed pg_regress binary to run the tests. 
We don't actually need to compile everything, just pg_regress. Adding it to gpdb_pitr tests Makefile similar to how isolation2 is compiled and installed.

**Pull request merged**